Runtime/MonkeyConfig.cs

```diff
@@ -49,10 +54,19 @@ public class MonkeyConfig
         /// Function returns the screen position where monkey operators operate on for the specified gameObject
         /// </summary>
         public Func<GameObject, Vector2> ScreenPointStrategy = DefaultScreenPointStrategy.GetScreenPoint;
-        
+
+        /// <summary>
+        /// Function returns the random string generation parameters
+        /// </summary>
+        public Func<GameObject, RandomStringParameters>
+            RandomStringParametersStrategy = DefaultRandomStringParameterGen;
```

Comment:
IMO:
I think it is better to include it in RandomStringImpl If strategy is only one.
Or is it better to have a dictionary with a strategy for each TextField name?

Reply:
I didn't face it but I assumes there are possibly many strategy and we cannot decide what is a best. I can list at least 4 strategies:

- Name-based strategy (as you proposed)
- Hierarchy-based strategy
- Annotation-based strategy
- Mixed strategy above
- ...

So, I think it is better that keep extensibility about it.